uxarray/helpers.py

```diff
@@ -1,7 +1,10 @@
-import os
+import math
```

Comment:
Is there a reason to use the math methods instead of the numpy methods? It seems like this package will pass around a bunch of numpy arrays so it might be better to just use the numpy methods directly